packages/manager/modules/vps/src/dashboard/migrate/migrate-confirm/migrate-confirm.routing.js

```diff
@@ -13,6 +13,8 @@ export default /* @ngInject */ ($stateProvider) => {
       selectedPlan: /* @ngInject */ ($transition$) => {
         return $transition$.params().selectedPlan;
       },
+      getRebootLink: /* @ngInject */ ($state) => () =>
```

Comment:
question: can't we put `getRebookLink` on parent level instead of adding it inside each module. Otherwise looks good to me.

Reply:
the states `upgrade`, `migrate` are not the children of dashboard state. So it cannot be included in one place and reuse.